docs/src/examples/components/Text/Variations/index.tsx

```diff
@@ -4,6 +4,18 @@ import ExampleSection from 'docs/src/components/ComponentDoc/ExampleSection'
 
 const Variations = () => (
   <ExampleSection title="Variations">
+    <ComponentExample
+      title="TextAlign"
+      description={
+        <>
+          Text may be aligned to the left, right, center or be justified.
+          <br />
+          You need to use Text as a block element (<code>as='div'</code>) so that textAlign prop can
```

Comment:
Probably you already considered this, but why don't we just add `display: 'block'` in the styles if the textAlign prop is provided?

Reply:
Good that you opened this conversation. I have considered it briefly but thought it might be unexpected behavior for the user.

Giving it a second thought if a user adds `textAlign` they probably expect the `Text` component to behave as a block otherwise alignment does not make sense. So I think you're right here. Let me make the change and ask others to chip in as well: @layershifter @kuzhelov @miroslavstastny

Reply:
applied `display: 'block' ` when `textAlign` prop is provided